admin/metabox/class-metabox.php

```diff
@@ -501,14 +501,17 @@ protected function get_additional_tabs() {
 	/**
 	 * Adds a line in the meta box.
 	 *
-	 * @todo [JRF] Check if $class is added appropriately everywhere.
+	 * @deprecated 23.5
+	 * @codeCoverageIgnore
 	 *
 	 * @param string[] $meta_field_def Contains the vars based on which output is generated.
 	 * @param string   $key            Internal key (without prefix).
 	 *
 	 * @return string
 	 */
 	public function do_meta_box( $meta_field_def, $key = '' ) {
+		_deprecated_function( __METHOD__, 'Yoast SEO 23.5' );
```

Comment:
There is also a filter below (`wpseo_do_meta_box_field_`). Shouldn't we also deprecate that?

Reply:
Good catch!